pkgs/leak_tracker/lib/src/leak_tracking/_object_record.dart

```diff
@@ -26,6 +28,8 @@ class ObjectRecord {
   /// the object is already GCed and thus there is no access to its code.
   final IdentityHashCode code;
 
+  final CreationChecker? creationChecker;
```

Comment:
dart doc. Prefer to add dart doc for all of these public members while you are here.